cachet-tools/Dockerfile

```diff
@@ -1,21 +1,34 @@
 # Get centos base image
 # Use FROM scratch for blank image
-FROM python:2
+FROM docker.io/centos/python-36-centos7
 
 # who are you
-MAINTAINER bennyturns
+MAINTAINER aarif
 
-COPY requirements.txt /
+# copy file into place
+COPY update_cachet.py /
+COPY services.py /
+COPY hosted-services.json / 
+COPY populate-cachet.py /
+COPY purge-cachet.py /
+COPY requirements.txt / 
+CMD ["ls -l /"]
```

Comment:
is this useful to have?

Reply:
Yes, left in there for debugging purpose.